packages/ssx-server/src/middlewares/express/endpoints.ts

```diff
@@ -66,8 +67,9 @@ const ssxEndpoints = (ssx: SSXServer) => {
       req.session.siwe = session.siwe;
       req.session.signature = session.signature;
       req.session.daoLogin = session.daoLogin;
+      req.session.ens = session.ens;
 
-      res.status(200).json({ session: req.session });
+      res.status(200).json({ ...req.session });
```

Comment:
This is an API change (backwards incompatible?)

Reply:
Yes, it is a potential breaking change. The `/ssx-login` endpoint is supposed to return the session. But it was returning a session object with a session property (`session.session`)  in this first version. This change fixes it to only send the session with all information, but it can break if someone was trying to access `session.session`.

Reply:
I think this the right approach! Even as a breaking change, it is a better API.